sdk/src/DocuSign.Admin/Api/AccountsApi.cs

```diff
@@ -342,10 +342,16 @@ public ApiResponse<MemberGroupsResponse> GetGroupsWithHttpInfo(Guid? organizatio
                 if (options.end != null) localVarQueryParams.Add("end", this.ApiClient.ParameterToString(options.end)); // query parameter
             }
 
+            // authentication (docusignAccessCode) required
```

Comment:
@garg-mudit This is getting added in multiple places now.

Reply:
Ticket [DCM-11094](https://docusign.atlassian.net/browse/DCM-11094) is present to check the same. 
As the code is not a breaking change, thus its okay keep it for now.